runtime/packages/core-utils/assembly/utils.ts

```diff
@@ -38,7 +38,7 @@ export namespace Utils {
         let result: u8[] = [];
         for (let i=0; i<args.length ; i++){
             const strScale = new ScaleString(args[i]); 
-            result = result.concat(strScale.toU8a());
+            result = result.concat(strScale.values);
```

Comment:
don't you need to get the u8 bytes?

Reply:
ScaleString encodes the length of the string in the beginning, i.e `"data" => [16, 100, 97, 111, 97]` and `ext_logging_log_version_1` doesn't expect SCALE encoded string, so 16 in the beginning is converted to character as well